plugins/snapshots/erofs/erofs.go

```diff
@@ -393,6 +436,10 @@ func (s *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 		}
 		td = ""
 
+		if !strings.Contains(key, snapshots.UnpackKeyPrefix) {
+			s.generateFsMeta(ctx, snap.ParentIDs)
```

Comment:
Is it possible to handle this outside the transaction, in case the command gets stuck?

Reply:
it looks like it's nice to have. so it should be safe to move it outside the transaction

Reply:
done, please take another look...